modules/core/src/main/java/org/apache/ignite/spi/communication/tcp/TcpCommunicationSpi.java

```diff
@@ -353,38 +249,19 @@
     public static final int DFLT_MSG_QUEUE_LIMIT = GridNioServer.DFLT_SEND_QUEUE_LIMIT;
 
     /**
-     * Default count of selectors for TCP server equals to
-     * {@code "Math.max(4, Runtime.getRuntime().availableProcessors() / 2)"}.
+     * Default count of selectors for TCP server equals to {@code "Math.max(4, Runtime.getRuntime().availableProcessors()
+     * / 2)"}.
      */
     public static final int DFLT_SELECTORS_CNT = Math.max(4, Runtime.getRuntime().availableProcessors() / 2);
 
-    /** Default initial connect/handshake timeout in case of failure detection enabled. */
-    private static final int DFLT_INITIAL_TIMEOUT = 500;
-
-    /** Default initial delay in case of target node is still out of topology. */
-    private static final int DFLT_NEED_WAIT_DELAY = 200;
-
-    /** Default delay between reconnects attempts in case of temporary network issues. */
-    private static final int DFLT_RECONNECT_DELAY = 50;
-
-    /**
-     * Version when client is ready to wait to connect to server (could be needed when client tries to open connection
-     * before it starts being visible for server)
-     */
-    private static final IgniteProductVersion VERSION_SINCE_CLIENT_COULD_WAIT_TO_CONNECT = IgniteProductVersion.fromString("2.1.4");
-
     /** Connection index meta for session. */
     public static final int CONN_IDX_META = GridNioSessionMetaKey.nextUniqueKey();
 
     /** Node consistent id meta for session. */
     public static final int CONSISTENT_ID_META = GridNioSessionMetaKey.nextUniqueKey();
 
-    /** Message tracker meta for session. */
-    private static final int TRACKER_META = GridNioSessionMetaKey.nextUniqueKey();
-
     /**
-     * Default local port range (value is <tt>100</tt>).
-     * See {@link #setLocalPortRange(int)} for details.
+     * Default local port range (value is <tt>100</tt>). See {@link #setLocalPortRange(int)} for details.
```

Comment:
After change, javadoc become less readable and still spans over 3 lines.
Please either make in true oneliner or revert change.